types/rgb2hex/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for rgb2hex 0.2
+// Project: https://github.com/christian-bromann/rgb2hex
+// Definitions by: abh80 <https://github.com/abh80>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare function hex2rgb(rgb: string): {
+	hex: string;

```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/rgb2hex/index.d.ts;
  Line: 6;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Allright ! Thank you